.circleci/config.yml

```diff
@@ -141,6 +151,12 @@ jobs:
               export CF_PASSWORD="$CF_PASSWORD_DEV"
               export CGHOSTNAME_BACKEND="$CGHOSTNAME_BACKEND_DEV"
               export CGHOSTNAME_FRONTEND="$CGHOSTNAME_FRONTEND_DEV"
+            elif [ "$CIRCLE_BRANCH" == "raft-staging" ] ; then
```

Comment:
We needed to include this to test the automated deploy to staging. Do we want this in the final version? We have been talking about deploying to `development` by other methods and deploying to `vendor-staging` when we merge to `raft-tdp-main`. Would love to get your thoughts @ADPennington and @alexsoble

Reply:
Personally, I think it would be better to deploy to `vendor-staging` anytime a merge is made to `raft-tdp-main` and then deploy to `development` based on some manual trigger that can spark off a build in CircleCI (like a PR comment or adding a label). Until the work is done to get trigger-able deployments we could use `raft-review` in the meantime to deploy to development.

I'll defer to @ADPennington and @alexsoble for final decision though.

Reply:
My two cents, I like the idea of deploying to `vendor-staging` based off a merge to `raft-tdp-main` and to `development` based off a merge to `raft-review` (for now!). Reuse the branches we already have now. Agreed that trigger-able deployments for dev would be a nice future improvement.